apps/cli/src/commands/reset.command.ts

```diff
@@ -0,0 +1,134 @@
+import type {
+  CommandActionData,
+  CommandOption
+} from '@/types/command/command.types'
+import BaseCommand from '@/commands/base.command'
+import { confirm, log } from '@clack/prompts'
+import { handleSIGINT } from '@/util/prompt'
+import os from 'os'
+import path from 'path'
+import { access, chmod, mkdir, writeFile } from 'fs/promises'
+import { constants } from 'fs'
+
+const KEY_PERMISSIONS = 0o600
+
+export default class ResetCommand extends BaseCommand {
+  getName(): string {
+    return 'reset'
+  }
+
+  getDescription(): string {
+    return 'Reset local Keyshade CLI profiles to empty objects'
+  }
+
+  getOptions(): CommandOption[] {
+    return [
+      {
+        short: '-y',
+        long: '--yes',
+        description: 'Skip interactive confirmation',
+        defaultValue: false
+      },
+      {
+        short: '-d',
+        long: '--dry-run',
+        description: 'Show what would be reset without making changes',
+        defaultValue: false
+      }
+    ]
+  }
+
+  async action({ options }: CommandActionData): Promise<void> {
+    const yes = Boolean(options.yes ?? options.Y ?? options.y)
+    const dryRun = Boolean(options['dry-run'] ?? options.dryRun ?? options.d)
+
+    if (!yes) {
+      const shouldContinue = await confirm({
+        message: 'This will erase all locally saved profiles. Continue? (y/N)'
+      })
+      handleSIGINT(shouldContinue, 'Reset cancelled!')
+
+      if (!shouldContinue) {
+        log.info('Reset cancelled. No changes were made.')
+        return
+      }
+    }
+
+    const homeDirectory = os.homedir()
+    const keyshadeDirectory = path.join(homeDirectory, '.keyshade')
+    const targets = [
+      {
+        fileName: 'default-profile.json',
+        displayName: 'Default profile',
+        filePath: path.join(keyshadeDirectory, 'default-profile.json')
+      },
+      {
+        fileName: 'profiles.json',
+        displayName: 'Profiles',
+        filePath: path.join(keyshadeDirectory, 'profiles.json')
+      }
+    ]
+
+    if (!dryRun) {
+      await this.ensureDirectory(keyshadeDirectory)
+    }
+
+    for (const target of targets) {
+      const exists = await this.fileExists(target.filePath)
```

Comment:
For the file operations, check the `configuration.ts` file. It already has all the file ops you would need. What you would need to use are `writeProfileConfig` and `writeDefaultProfileConfig` functions.